File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Cpp/pattern1.cpp
/*
Program 1: Write a Program to Print following Pattern
1
2 2 
3 3 3
4 4 4 4
*/

#include<iostream>


int main(){

	for(int outer = 1 ; outer <=4; outer++ ){

		for(int inner = 1 ; inner <=outer; inner++){
			
			printf("%d",outer);
		}		
		printf("\n");
	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Java/Pattern1.java
/*
Program 1: Write a Program to Print following Pattern
1
2 2 
3 3 3
4 4 4 4
*/

class Patter1{
	
	public static void main(String args[]){

		for(int outer = 1 ; outer <=4; outer++ ){

			for(int inner = 1 ; inner <=outer; inner++){
			
				System.out.printf("%d",outer);
			}		
			System.out.printf("\n");
		}
	}	

}	
